rust/routee-compass-macros/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "routee-compass-macros"
+version = "0.1.0"
```

Comment:
should this match the versions of the rest of the routee crates?

Reply:
yeah that probably makes the most sense